docs/mapview.md

```diff
@@ -50,6 +50,7 @@ To access event data, you will need to use `e.nativeEvent`. For example, `onPres
 | `onKmlReady` | `KmlContainer` | Callback that is called once the kml is fully loaded.
 | `onRegionChange` | `Region` | Callback that is called continuously when the region changes, such as when a user is dragging the map.
 | `onRegionChangeComplete` | `Region` | Callback that is called once when the region changes, such as when the user is done moving the map.
+| `onIdleCamera` | `{ bounding: { lat_max: Number, lon_max: Number, lat_min: Number, lon_min: Number,}, Region }` | Callback that is called once when the camera is idle at its position, such as when the user is done moving the map.(Only for GoogleMaps)
```

Comment:
Why this notation and not the same with `getMapBoundaries`?

Reply:
I feel working with max and min is much easier and robust than using northEast and southWest. With the Map rotation there will be no confusion for direction and it can easily be figured out with max/min

Reply:
@alvelig @christopherdro any thoughts on this? I tend to go to consistent APIs but maybe you have another opinions?

Reply:
Yes, I'm also pro Region object format rather than introducing not consistent options. Just because of the custom.
Region is easier to treat when zoom (you change 2 parameters instead of 4 and you don't have to calculate center position in that case)

Reply:
Region parameter is also being passed along with the bounding areas. Bounding area has been added to get the exact lat and lon min/max values which helps in many api requests. There are many PR for such values so I though of  keeping it with same name.

Reply:
@ShivamKJJW I agree with you that sometimes it would be easier to use this notation (I also had to deal with it a few times).
However we already have some inconsistencies between the APIs so I wouldn't introduce another one. Somebody who's using region probably already has the adapter in place if they need to convert to X notation. 
Adding a new one would add an extra layer of complexity for everybody - old users would need to re-adapt, new users will be confused by different notations.

As an aside, there is  no good or bad notation, is just a matter of conventions 😀

So it would be amazing if you'd want to move to region as it is now ❤️

Reply:
@rborn Right yeah! I will change the things and push it when I get time. 😸